docs/readmes/lte/suci_extensions.md

```diff
@@ -0,0 +1,92 @@
+
+The encryption and decryption mechanism specified in TS 33.501 is shown below.
+
+- According to 3GPP TS 33.501, the UE generates SUCI with the public key of the home network using an ECIES-based protection scheme.
+- According to 3GPP TS 33.501 clause C.3.2 (Processing on the UE side), the UE shall use the home network's provisioned public key and construct a new ECC (Elliptic Curve Cryptography) ephemeral public/private key pair based on the ECIES parameters provisioned by the home network.
+- According to 3GPP TS 33.501 clause C.3.3 (Processing on the home network side), the home network must use the UE's received ECC ephemeral public key and the home network's private key to de-conceal an SUCI.
+
+## Architecture
+
+Subscriberdb of AGW shall periodically fetch the list of SuciProfiles from Orc8r. Based on the home network public key identifier received in the authentication request, corresponding private key is retrieved and used to de-conceal the SUCI in the authentication request. Each public/private key pair is not subscriber specific, and can be shared among multiple subscribers.
+
+Assuming public/private key pair generation of HPLMN is not part of magma, operator shall configure list of SuciProfiles in AGW which comprises of
+
+- Home Network Public Key Identifier
+- Public/Private Key pair
+- Protection scheme
+
+Operator is responsible for provisioning UE with Home Network Public Key Identifier, Home Network Public key, Protection scheme.
+
+## Detailed Design
+
+![Design of SUCI Extensions Network Flow](assets/lte/design_of_suci_extensions.png?raw=true "Design of SUCI Extensions Network Flow")
+
+The diagram above represents the network create/update flow and the modules involved in processing the NMS request. Also represents how the SuciProfiles are fetched from Orc8r.
+
+### Orc8r
+
+- To facilitate REST API updates, the LTE Network data model will accept a list of SuciProfiles and obsidian handlers.
+- Subscriberdb RPC servicer to handle the gRPC request from AGW, which populates the list of SuciProfiles per network.
+
+### AGW
+
+- AMF to invoke SIDF Module of subscriberdb for SUCI decryption.  In initial registration NAS message, if type of identity is SUCI (1) and protection scheme Id is either ECIES scheme profile A OR ECIES scheme profile B, then AMF initiates GRPC call to subscriberdb with parameters such as Home network public key identifier, ue ciphertext and encrypted imsi.
+- SubscriberDB periodically fetches the list of SuciProfiles from Orc8r and stores it as in- memory data.
+- SubscriberDB has SIDF functionality, that performs an in-memory data lookup to extract the private key associated with the home network public key identifier and protection scheme that we obtained from the initial registration NAS message. SIDF has an algorithm that uses the private key to decode the SUCI.
+
+## Prerequisites
+
+Prior to configuring SUCI extensions, you must first support integrated 5G SA FWA.

```

Comment:
`5G SA FWA` is used here the first time. Give the full term for the acronym, please.

Reply:
Addressed Review Comment